### PR TITLE
Modified exec_command to understand multi option command put "ssl" next last command.

### PR DESCRIPTION
**Pull request by fakeskimo**
"ssl" is need to put just next of second command part when using current version lgtv [commit link](https://github.com/klattimer/LGWebOSRemote/commit/514be54225986aa2c14f6fd12ddef7bbc4378cc1) 

Example:
lgtv MyTV startApp com.webos.app.hdmi1 ssl --> fails
lgtv MyTV startApp ssl com.webos.app.hdmi1 --> success

Therefore I added step to put "ssl" next of second command part in exec_command.

### CODE REVIEW

**Review by cmer — CHANGES_REQUESTED** [review]
This is awesome thank you. Could you put back the `lgtv_disabled` function before I approve, however?

**Comment by fakeskimo** [comment]
I put back original "lgtv_disabled" function to resolve the conflict. LMK if there is any further issue. Thanks!

**Comment by zakkhoyt** [comment]
@fakeskimo YES! Great catch. I have been trying to diagnose this problem today. It hadn't occurred that the args were out of order.

File: lgtv_init.lua [hunk 1]
```diff
@@ -52,7 +52,13 @@ end
 
 function exec_command(command)
   if lgtv_ssl then
-    command = command.." ssl"
+    space_loc = command:find(" ")
+
+    if space_loc then
```
Comment (zakkhoyt) [inline lgtv_init.lua:57]:
```suggestion

    --- "ssl" must be the first argument for a command. Advance it to the expected position.
    if space_loc then
```

**Review by cmer — APPROVED** [review]

**Pull request merged**